dump/src/reader/compat/v5_to_v6.rs

```diff
@@ -405,11 +405,11 @@ impl From<v5::Action> for v6::Action {
             v5::Action::SettingsAll => v6::Action::SettingsAll,
             v5::Action::SettingsGet => v6::Action::SettingsGet,
             v5::Action::SettingsUpdate => v6::Action::SettingsUpdate,
-            v5::Action::StatsAll => v6::Action::StatsAll,
+            v5::Action::StatsAll => v6::Action::StatsGet,
             v5::Action::StatsGet => v6::Action::StatsGet,
-            v5::Action::MetricsAll => v6::Action::MetricsAll,
+            v5::Action::MetricsAll => v6::Action::MetricsGet,
             v5::Action::MetricsGet => v6::Action::MetricsGet,
-            v5::Action::DumpsAll => v6::Action::DumpsAll,
```

Comment:
@Kerollmops I'm not sure whether this will cause issues, we'd need a v6 to v7 if we do this?

Reply:
Hum... You are probably right. However, if the actions can be converted to string it's ok I think. @ManyTheFish told me that in the dumps everything was converted to strings.

Reply:
yes, but it may be possible to create a dump v6 containing `v6::Action::MetricsAll` or `v6::Action::StatsAll`, which is removed in this branch; it may crash, no?

Reply:
Yeah, I think so too. If I'm reading it right, currently it uses `serde_json` to write the dumps, and with `metrics.*` and the others removed, if we try and import an older dump it should crash.

There are two options:
1. Create new dump version
2. Somehow make `Sequence` impl work without removing any flags

I'll try to make 2. work.